methods/methods_release2.md

```diff
@@ -93,49 +95,262 @@ Demultiplexing, feature-table generation, fragment insertion, and taxonomic prof
 * Process: Closed-reference OTU picking 
 * Parameters: 97% sequence similarity threshold, using the GreenGenes 13_8 release as a reference
 
-
 ### 1.2 18S rRNA gene data
 
 #### Demultiplexing:
-* Process: Split libraries FASTQ
-* Parameters: Multiplexed FASTQ, Golay 12 base pair reverse complement mapping file barcodes
+`qiime demux emp-paired \
+  --i-seqs emp-paired-end-sequences.qza \
+  --m-barcodes-file emp500_prep_info_18S.txt \
+  --m-barcodes-column 'barcode' \
+  --p-rev-comp-mapping-barcodes \
+  --o-error-correction-details emp500_18s_demux.qza \
+  --o-per-sample-sequences emp500_18s_demux.qza`
+
+#### Trimming primers:
+`qiime cutadapt trim-paired \
+  --i-demultiplexed-sequences emp500_18s_demux.qza \
+  --p-front-f GTACACACCGCCCGTC \
+  --p-adapter-f GTAGGTGAACCTGCAGAAGGATCA \
+  --p-front-r GACGGGCGGTGTGTAC \
+  --p-adapter-r TGATCCTTCTGCAGGTTCACCTAC \
+  --o-trimmed-sequences emp500_18s_demux_trimmed_1of3.qza`
+
+`qiime cutadapt trim-paired \
+  --i-demultiplexed-sequences emp500_18s_demux_trimmed_1of3.qza \
+  --p-front-f TTAGTGAGGCCCT \
+  --p-adapter-r CCAATCGGTAGTAGCGACGGGC \
+  --o-trimmed-sequences emp500_18s_demux_trimmed_2of3.qza`
+
+`qiime cutadapt trim-paired \
+  --i-demultiplexed-sequences emp500_18s_demux_trimmed_2of3.qza \
+  --p-adapter-r AGGGCCTCACTAA \
+  --o-trimmed-sequences emp500_18s_demux_trimmed_3of3.qza`
+
+#### Export data and process for denoising with Deblur (standalone)
+Note: The sample list is a text file where each row is a unique sample identifier that corresponds to your sequence file names. For example, 'uren.s001' corresponds to the sequence files 'uren.s001_S1_L001_R1.fastq.gz' and 'uren.s001_S1_L001_R2.fastq.gz'.
+
+`qiime tools export \
+  --input-path emp500_18s_demux_trimmed_3of3.qza \
+  --output-path emp500_18s_demux_trimmed`
+  
+`input_path='emp500_18s_demux_trimmed/'`
+`output_path='emp500_18s_demux_trimmed_merged_files/'`
+`sample_list='sample_list.txt'`
 
-#### Sequence trimming, denoising, feature-table generation, and fragment insertion:
-* Process: Trimming
-* Parameters: 150 base pair
-* Process: Deblur
-* Parameters: Default settings
+`for i in $(cat < "$sample_list");
+do
+cat "$input_path""$i"*R1*fastq.gz "$input_path""$i"*R2*fastq.gz > "$output_path""$i".fastq.gz
+done`
 
-#### Taxonomic profiling
-* Process: Feature-classifier sklearn 
-* Parameters: Using the SILVA 138.1 release as a reference
+#### Denoising
```

Comment:
It might be nice to add a comment that it was necessary to run outside of qiime2 to gain access to the all.biom table

Reply:
Thanks! This is made clear on line 129 above.